tests/source-maps.test.js

```diff
@@ -38,33 +38,34 @@ test.skip('apply generates source maps', async () => {
   // All CSS generated by Tailwind CSS should be annotated with source maps
   // And always be able to point to the original source file
   expect(sources).not.toContain('<no source>')
-  expect(sources.length).toBe(1)
+  expect(sources.length).toBe(2)
 
-  expect(annotations).toEqual([
-    '2:6 -> 2:6',
-    '3:8-29 -> 3:8-29',
-    '4:8-35 -> 4:8-20',
-    '4:8-35 -> 5:8-19',
-    '4:8-35 -> 6:8-26',
-    '4:8-35 -> 7:8-63',
-    '5:6 -> 8:6',
-    '7:6 -> 10:6',
-    '8:8-41 -> 11:8-41',
-    '9:8-33 -> 12:8-20',
-    '9:8-33 -> 13:8-19',
-    '9:8-33 -> 14:8-26',
-    '9:8-33 -> 15:8-63',
-    '10:6 -> 16:6',
-    '13:8 -> 18:6',
-    '13:8-31 -> 19:8-20',
-    '13:8-31 -> 20:8-19',
-    '13:8-31 -> 21:8-26',
-    '13:8 -> 22:8',
-    '13:31 -> 23:0',
-  ])
+  expect(annotations).toMatchInlineSnapshot(`
```

Comment:
Now that we don't use `crosscheck` anymore, we can use `toMatchInlineSnapshot` again!

Reply:
🎉🎉🎉